crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -117,8 +131,10 @@ async fn create(
         value,
         schema,
         function_name,
-        created_by: user.get_email(),
-        created_at: Utc::now(),
+        created_by: created_by_val,
+        created_at: created_at_val,
+        last_modified_at: Utc::now().naive_utc(),
```

Comment:
```suggestion
        last_modified_at: created_at_val,
```
These two should be the same right? Lets not recompute the time here

Reply:
no @Datron , created_at_val can be from already existing row, thats why recomputed the time

Reply:
@pratikmishra356 this is inside create function, how can created_at_val previously exist? Either way, the mark is for this row right? created_at should be recomputed and the same should be used for last_modified_at

Reply:
@Datron the handler name is create , but we have only one api for create or update/ the handler name is misguiding 
we create default_config struct from combination of req data and existing data , 
`let result = fetch_default_key(&key, &mut conn);`